docs/guides/runners.md

```diff
@@ -0,0 +1,114 @@
+# Running benchmarks with the benchmark runner
+
+The `BecnhmarkRunner` is `nnbench`'s abstract interface to collect and execute benchmarks.
+You can either use it directly or instantiate a derivative class to override selected methods and implement you own logic.
```

Comment:
I would drop this, since it is a no-content sentence (this is true for almost every Python type).